src/domain/tradingview/utils.ts

```diff
@@ -0,0 +1,13 @@
+export const supportedResolutions = { 5: "5m", 15: "15m", 60: "1h", 240: "4h", "1D": "1d" };
+export const LAST_BAR_REFRESH_INTERVAL = 15000; // 15 seconds
+
+export function getPeriodFromResolutions(value, object = supportedResolutions) {
```

Comment:
object -> resolutions?

value - what's value?) if it's resolution, maybe call it "resolution"?

Reply:
Renamed